public/app/common/components/cc-form/cc-form.jsx

```diff
@@ -0,0 +1,36 @@
+var React = require('react');
+var styleMixin = require('mixins/style-mixin');
+var t = require('tcomb-form');
+var Form = t.form.Form;
+
+var options = {
+  templates: {
+    textbox: require('./ccTextInput.jsx')
+  }
+};
+
+var CCForm = React.createClass({
```

Comment:
Here t's not so straightforward, because we can get name clashes. We already have Form component from library, so we need to configure it and put some styles on it. 

I would say that cc-form component is usable only in context of control center